kernel/libs/xarray/src/test.rs

```diff
@@ -0,0 +1,335 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use alloc::{boxed::Box, sync::Arc};
+use core::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
+
+use ostd::{
+    prelude::ktest,
+    task::{disable_preempt, Task},
+};
+
+use crate::*;
```

Comment:
```suggestion
use super::*;
```

Or just explicitly list what items get imported. I don't think there are many.